go/vt/schemadiff/types.go

```diff
@@ -94,6 +94,11 @@ const (
 	TableCharsetCollateIgnoreAlways
 )
 
+const (
+	TableQualifierUseSpecified int = iota
+	TableQualifierUseOriginal
+)
```

Comment:
@rohit-nayak-ps Hmm, doesn't this change the default behavior if `TableQualifierUseSpecified` is the `0` option? Shouldn't these be flipped to not change the default behavior and only explicitly opt into this?

Reply:
Those enums are not intuitive: was going to ask for better naming. That is currently the default that is already in `schemadiff`: it uses the qualifier of the desired (`to`) schema. So essentially `UseOriginal` is the one in `from`.